django-verdant/rca/templates/rca/base.html

```diff
@@ -116,6 +116,17 @@
             <p class="chromeframe">You are using an <strong>outdated</strong> browser which this site does not fully support. Please <a href="http://browsehappy.com/">upgrade your browser</a> or <a href="http://www.google.com/chromeframe/?redirect=true">activate Google Chrome Frame</a> to improve your experience.</p>
         <![endif]-->
 
+        <div class="survey-banner" id="#survey-banner">
+            <div class="survey-banner__content">
+                <div class="survey-banner__image-wrapper">
+                    <img class="survey-banner__image" src="{% static "rca/images/amzn100.png" %}" />
+                </div>
+                <p class="survey-banner__text">Give us feedback <span class="survey-banner__dash">- &nbsp;</span><span class="survey-banner__bold-text">receive a £10 Amazon.co.uk Gift Card.</span></p>
+                <a href="https://surveymonkey.com" class="button survey-banner__button">Participate</a>
+            </div>
+            <a id="survey-banner-close" class="survey-banner__close" href="#">close</a>
```

Comment:
Could this be a `button` instead of `a` so it’s not announced as a link to screen reader users?

Reply:
I might leave this as a link now unless you think this is really detrimental - the site has a load of button styles which are tedious to remove.

Reply:
👍 I’ll let you be the judge. It’s detrimental to the user experience but not to a point of making it unusable.